content/en/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md

```diff
@@ -7,7 +7,7 @@ description: Configure the kubelet's image credential provider plugin
 content_type: task
```

Comment:
In `content/en/docs/concepts/containers/images.md` under `## Using a private registry` add the following reference to this doc:

```
### Kubelet Credential Provider

{{< note >}}
This approach is suitable when kubelet needs to fetch registry credentials dynamically.
Most commonly used for registries provided by cloud providers where auth tokens are short-lived. 
{{< /note >}}

The Kubelet can be configured to invoke a plugin binary to dynamically fetch registry credentials for a container image.
This is the most robust and versatile way to fetch credentials for private registries, but also requires kubelet-level configuration to enable.

See the [Kubelet Credential Proivder](./content/en/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md) docs for more details.      
```

(or something like that)